terraform/aks/config/review.tfvars.json

```diff
@@ -0,0 +1,11 @@
+{
+  "environment": "review",
+  "file_environment": "review",
+  "app_key_vault_name": "s189t01-gitapi-rv-app-kv",
+  "infra_key_vault_name": "s189t01-gitapi-rv-inf-kv",
+  "cluster": "test",
+  "namespace": "git-review",
+  "enable_monitoring": false,
+  "enable_logit": true,
+  "enable_prometheus_monitoring": true
```

Comment:
review envs must default to "deploy_azure_backing_services": false
as they default to container postgres and redis